docs/developers-guide.md

```diff
@@ -3,319 +3,130 @@
-
-It's generally more recommended to use `make upload-images` to build and push image.
-But you might need to build image and run test on it if you're running a test job to verify that image.
-
-To prepare default environment variables:
-
-```
-# default value of VERSION is "{part of latest commit id}-dirty"
-export VERSION=latest
+### Troubleshooting
+* Show verbose information for openstack API request
 
-# default value for IMAGE_NAMES includes all supported images.
-export IMAGE_NAMES='openstack-cloud-controller-manager magnum-auto-healer'
-# default value for ARCHS includes all supported architectures.
-```
+  To get better understanding about communication with openstack, you can set `--v=6` to see more detailed API requests and responses. It is recommended to set `--concurrent-service-syncs=1` as well.
 
-If you not plan to build image, you can directly run `make images-all-archs` to support build for all specified images with all specified architectures.
+### Review process
+Everyone is encouraged to review code. You don't need to know every detail of the code base. You need to understand only what the code related to the fix does.
 
-```
-export ARCHS='amd64 arm arm64 ppc64le s390x'
-make images-all-archs
-```
+As a reviewer, remember that you are providing a service to submitters, not the other way around. As a submitter, remember that everyone is subject to the same rules: even the repo maintainers voting on your patch put their changes through the same code review process.
 
-If you plan to build image for single architecture, you can either specify `ARCHS` as `amd64` or run below commands
+Read [reviewing changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests) for how the github review process looks like.
 
-```
-export ARCH=amd64
-make build-images
-```
-
-Or if you wish to ony build for specific architecture with specific image
-
-```
-export ARCH=amd64
-make image-openstack-cloud-controller-manager
-```
-In case of building images, by default, images built are appended with this -$ARCH tag at the end.
-For example, for the above case, openstack-cloud-controller-manager-amd64 is built.
-In case needed without it, tag it separately as required.
-
-```
-docker image tag openstack-cloud-controller-manager-amd64:latest openstack-cloud-controller-manager:latest
-```
+Guidance for the code reviewer:
 
-This will create image with tag `openstack-cloud-controller-manager:latest` in your local environment.
+* We need at least two `/lgtm` from reviewers before `/approve` a PR. However, two `lgtm` is not equal to a `/approve`.
+* Only the people in the `approvers` section of the OWNERS file have the `/approve` permission.
+* In certain circumstances we allow `/approve` with only one `/lgtm` such as:
+  * Typo fix
+  * CI job quick fix
+  * Changing document with no more than 3 lines
+* If a PR needs further discussion and to avoid unintentional merge, use `/hold`.
```

Comment:
May be we could add , self /lgtm should be avoided

Reply:
I seem to do `/lgtm` to my own PR before but I remember can't do that. I can try again in this PR later on.